### PR TITLE
Adding /purgeparticles [Formatting]

### PR DESCRIPTION
**Pull request by ghost**
I've requested Prozza for once server.DispatchCommand using
I hope he'll aggree

It is a command for purge players ./pp not /particle, /purgeall wipes /particle it wipes the ParticlePlugin /pp  

Please don't comment: It is denied cause you used Dispatch Command. Wait for Prozza, please

### CODE REVIEW

**Comment by JeromSar** [comment]
@Adamx120 I have no idea what you're talking about... Is this an issue for the /purgeparticles command? I think you really need to read the [Contributing Guidelines](https://github.com/TotalFreedom/TotalFreedomMod/blob/master/CONTRIBUTING.md)

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_purgeparticles.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import org.bukkit.entity.Player;
+
+@CommandPermissions(level = AdminLevel.SUPER, source = SourceType.BOTH)
+@CommandParameters(description = "Clearing all particle effect from all players on the server", usage = "/<command> <partialname>")
+public class Command_purgeparticles extends TFM_Command
+{
+    @Override
+    public boolean run(CommandSender sender, Player sender_p, Command cmd, String commandLabel, String[] args, boolean senderIsConsole)
+
+        {
```
Comment (rsajeey) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_purgeparticles.java]: formatting.

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_purgeparticles.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package me.StevenLawson.TotalFreedomMod.Commands;
+
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+@CommandPermissions(level = AdminLevel.SUPER, source = SourceType.BOTH)
+@CommandParameters(description = "Clearing all particle effect from all players on the server", usage = "/<command> <partialname>")
```
Comment (rsajeey) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_purgeparticles.java]: <partialname> isn't needed here.

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_purgeparticles.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+public class Command_purgeparticles extends TFM_Command
+{
+    @Override
+    public boolean run(CommandSender sender, Player sender_p, Command cmd, String commandLabel, String[] args, boolean senderIsConsole)
+
+        {           
+            {
+                for (Player player : server.getOnlinePlayers())
+                {
+                server.dispatchCommand(player, "pp clear");
```
Comment (rsajeey) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_purgeparticles.java]: server.dispatchCommand() isn't allowed as stated in https://github.com/TotalFreedom/TotalFreedomMod/blob/master/CONTRIBUTING.md

**Comment by Adamx120** [comment]
@JeromSar I created an issue

**Comment by JeromSar** [comment]
@Adamx120 You're making very little sense. I think you should refrain from making any more PRs, issues or commenting at all until you understand what you are doing, and how open source development works.

**Pull request closed without merging**